pkgs/by-name/ta/tagspaces/package.nix

```diff
@@ -0,0 +1,99 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  dpkg,
+  makeWrapper,
+  electron,
+  autoPatchelfHook,
+  libgcc,
+  glib,
+  nss,
+  at-spi2-atk,
+  cups,
+  libdrm,
+  gtk3,
+  mesa,
+  alsa-lib,
+  libGL,
+  libnotify,
+  libsecret,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tagspaces";
+  version = "5.9.2";
+
+  src = fetchurl {
+    url = "https://github.com/tagspaces/tagspaces/releases/download/v${version}/tagspaces-linux-amd64-${version}.deb";
```

Comment:
there's also a mac version available, too, e.g. https://github.com/tagspaces/tagspaces/releases/download/v6.1.0/tagspaces-mac-arm64-6.1.0.dmg

Reply:
Please read the above comment (https://github.com/NixOS/nixpkgs/pull/336094#issuecomment-2449742673). I always appreciate feedback, but this is useless if it doesn't build (from source)

Reply:
I am not sure I follow - I understand it is preferred to build from source, but there are other packages (https://github.com/NixOS/nixpkgs/blob/a833a6d94a8e6224e8e6c09ed3d47860665f67ee/pkgs/by-name/mo/mos/package.nix#L11 - this is the first match that came up when I searched, it wasn't picked because I know it) which use the dmg to enable use. We could use this until we can build it in the future or is that unacceptable?

Reply:
Sure, absolutely. We would need a version for Mac and one for Linux. This PR just tried to build it from source (and failed). But as an interim solution a prebuilt package would be an alternative 👍